watch-history-exporter-for-amazon-prime-video.js

```diff
@@ -1,3 +1,22 @@
+function germanDateToISO(germanDate) {
```

Comment:
Move this inside the function wrapper below and turn into an arrow function for consistency.